iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -85,6 +98,7 @@ class MainViewController: UIViewController, CardViewControllerDelegate, ThreedsW
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        UIFont.loadAllCheckoutFonts
```

Comment:
I am not sure we should ask the merchant implementing SDK to run this method in their app, as we do in our demo app.

I hope we can find a better way on this?

Reply:
This is to be registered in main app. the other one inside SDK.

Reply:
Is this a separate set of fonts that needs to be loaded? Do we have 1 set of fonts in the demo app, and a different set in the SDK?

Usually, this kind of environment building functionality would be inside the AppDelegate's `didFinishLaunching` lifecycle method?